examples/tv-casting-app/tv-casting-common/core/Attribute.h

```diff
@@ -239,12 +237,9 @@ class Attribute
     /**
      * @brief Subscribes to the value of the Attribute that belongs to the associated Endpoint and corresponding Cluster
      *
-     * @param context
      * @param successCb Called when the Attribute is read successfully, with the value of the attribute after reading, as well as
      * before (if the Attribute had been previously read)
      * @param failureCb Called when there is a failure in reading the Attribute
```

Comment:
why did we remove these? Looks like subscribe still takes in a context and min/max intervals?

Reply:
Added back with some documentation